front/components/spaces/UsedByButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  RobotIcon,
+  ScrollArea,
+} from "@dust-tt/sparkle";
+import type { DataSourceWithAgentsUsageType } from "@dust-tt/types";
+
+export const UsedByButton = ({
+  usage,
+  onItemClick,
+}: {
+  usage: DataSourceWithAgentsUsageType;
+  onItemClick: (assistantSid: string) => void;
+}) => {
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <Button
+          icon={RobotIcon}
+          variant="ghost"
+          isSelect
+          size="sm"
+          label={`${usage.count}`}
+        />
+      </DropdownMenuTrigger>
+      <DropdownMenuContent className="max-w-[300px]">
+        <ScrollArea className="border-1 max-h-[300px]">
```

Comment:
This won't work, `ScrollArea` needs a fixed size

Reply:
Thanks, fixed.